AppController/lib/zkinterface.rb

```diff
@@ -443,6 +443,14 @@ def self.get_cron_config(project_id)
     return JSON.load(cron_config_json)
   end
 
+  def self.get_datastore_servers()
+    return get_children('/appscale/datastore/servers').map { |server|
```

Comment:
Should there be some sort of contant here for the path? like /appscale should be our ZK_ROOT or something similar?